ostelco-ios-client/Models/Context.swift

```diff
@@ -19,7 +19,9 @@ struct Context: Codable {
     func getRegion() -> RegionResponse? {
 
         var ret: RegionResponse? = nil
-        var hasPendingStatus = false
+        
+        #warning("Figure out what this is for before deleting it entirely")
```

Comment:
@prasanthu @havard024 Any idea what this `hasPendingStatus` boolean is for?

Reply:
Lets remove it.

Reply:
Should be fine to remove it.
Some back story of this function, our app assumes that you only belong to one region while the APIs support multiple. Thus the APIs return a list and the app picks out the correct region, this function could easily just picked out the first region in the list